packages/core/admin/admin/src/pages/AuthPage/components/Register/tests/index.test.js

```diff
@@ -187,25 +218,4 @@ describe('ADMIN | PAGES | AUTH | Register', () => {
       message: expect.any(String),
     });
   });
-
-  it('Violates the yup schema and displays error messages', async () => {
```

Comment:
Why is this being removed?

Reply:
Previously the tests always passed because they were never setup using a proper yup schema - therefore the tests that were added by the community member would always have passed.

Now the Register component is always using the register schema for all tests, which makes it unnecessary to check for a custom schema.